api-docs/api/fuse/gestures/whilepressed/pressedposition.json

```diff
@@ -34,6 +34,10 @@
     },
     "syntax": {
       "uno": "public float2 PressedPosition { get; get; }"
+    },
+    "comment": {
+      "brief": "Getting the pressed position",
+      "full": "Get x and y from args, from rectPlaced function, and assign them to Observables.\n\n #Example\n\n    <WhilePressed ux:Name=\"wpressed\">\n  <Rectangle ux:Name=\"rect\" Position=\"{SnapshotProperty wpressed.PressedPosition}\" Placed=\"{rectPlaced}\" Anchor=\"0%,0%\" Width=\"20\" Height=\"20\" Color=\"#8f10\" />\n <Callback Handler=\"{onPressed}\" />\n</WhilePressed>"
```

Comment:
This is a generated file, and AFAICT, this *should* have been documented already, due to [this comment](https://github.com/fuse-open/fuselibs/blob/32cbbafd62e71284efafa7e600cafffb44e3beba/Source/Fuse.Gestures/Pressed.uno#L53). Aparently it doesn't appear in the docs, so I guess there's some generator bug here.

Anyway, the right solution to this is not to overwrite a generated file, as that will just get lost the next time we run the docs-generator.

Reply:
After a quick peek in the code @kusma links to, it looks like the doc-comment is assigned to the private field `_pressedPosition` and not the public property `PressedPosition`. 

The solution would probably be to move either the doc-comment or the private field, and regenerate the docs.

Reply:
Yeah, that makes sense.

Reply:
Sorry guys, I dunno how the docs work, perhaps ya could write a quick home page tut/getting started; would help everyone to contribute properly :)

Reply:
@kusma yeah, that's how I discovered how to use it, by looking into the source instead of the docs lol

Reply:
Fixed [here](https://github.com/fuse-open/fuselibs/pull/1259).